api-flask/docker-compose.override.yml

```diff
@@ -5,6 +5,7 @@ services:
       - ./app:/app
     ports:
       - "80:80"
+      - "443:80"
```

Comment:
I don't think this is useful? You don't want to run HTTPS in development anyway, and the complicated part of HTTPS isn't the port forwarding, it's the need to respond with an SSL certificate and encrypt the traffic. This doesn't actually achieve anything.